src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCharClass.MappingTable.cs

```diff
@@ -134,7 +132,7 @@ internal sealed partial class RegexCharClass
             new LowerCaseMapping('\u1FFA', '\u1FFB', LowercaseAdd, -126),
             new LowerCaseMapping('\u1FFC', '\u1FFC', LowercaseSet, 0x1FF3),
             new LowerCaseMapping('\u2160', '\u216F', LowercaseAdd, 16),
-            new LowerCaseMapping('\u24B6', '\u24CF', LowercaseAdd, 26),
+            new LowerCaseMapping('\u24B6', '\u24D0', LowercaseAdd, 26),
```

Comment:
It seems these changes do not affect IgnoringCase semantics or do they? If they do then the IgnoreCase BDD table needs regeneration, but this should be detected then by a failing unit test also.

Reply:
This is https://github.com/dotnet/runtime/pull/59425, in response to your https://github.com/dotnet/runtime/issues/58958.